src/screens/Dashboard/TransactionList.tsx

```diff
@@ -42,13 +43,18 @@ export class TransactionList extends Component<Props> {
     );
   };
 
+  onTransactionItemPress = (item: any) => {
+    // @ts-ignore
+    NavigationService.navigate('TransactionDetails', { item });
```

Comment:
why ts-ignore? Is it caused by `'TransactionDetails'`  instead of `Route.TransactionDetails`?

Reply:
I don't really know
`Argument of type '{ transaction: Transaction; }' is not assignable to parameter of type 'Partial<MessageProps> | Partial<EditTextProps> | WalletDetailsScreenProps | undefined'.
  Object literal may only specify known properties, and 'transaction' does not exist in type 'ScreenProps'.ts(2345)`